test/trace/test_trace.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from hpy.trace import call_counts, durations, set_trace_functions
+
+@pytest.fixture
+def hpy_abi():
+    return "trace"
+
+def _relative_call_count(name, call_count_0, diff):
+    return call_counts()[name] == call_count_0[name] + diff
+
```

Comment:
just a suggestion, you could make `call_count_0` a `collections.Counter()` and use use `.substract` to get the diff